vantage/business_metric_resource_test.go

```diff
@@ -58,6 +67,26 @@ func TestAccBusinessMetric_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("vantage_business_metric.test", "values.1.label", "a-label"),
 				),
 			},
+			{ // update values
```

Comment:
What should happen if I have an update of a resource that has values, but don't specify them in the update - is that captured?